ci-operator/config/ViaQ/vector/ViaQ-vector-v0.37.1-rh.yaml

```diff
@@ -61,8 +61,8 @@ images:
   to: logging-vector-test-unit
 promotion:
   to:
-  - name: "6.1"
-    namespace: logging
+  - namespace: logging
+    tag: v0.37.1
```

Comment:
Is `tag` correct here? I have only seen `name` so far. Loki is also using `name: v3.3.2`.

Reply:
One more: Should we keep the promotion to `6.1`, so that the builds that currently depend on this image still get it? Then we can do follow-up PRs to remove that dependency and, as a last one, remove the promotion?

Reply:
It is.  You can only use name or tag.  If you use tag, then the push to the registry uses buildname:tag instead of the openshift pattern which is the name spec'd here and the "tag" is the buildname.

Reply:
there is a version of 6.1 which already exists in the catalog so all existing builds will still pass

Reply:
> It is. You can only use name or tag. If you use tag, then the push to the registry uses buildname:tag instead of the openshift pattern which is the name spec'd here and the "tag" is the buildname.

Just had a look into the CI cluster and I think I need to migrate the Loki builds to that variant as well, it makes much more sense than using the version as the name like in the other pattern :+1: